lib/tmux_adapter.sh

```diff
@@ -1,12 +1,32 @@
 # Get the current path in the segment.
-get_tmux_cwd() {
-	local env_name=$(tmux display -p "TMUXPWD_#D" | tr -d %)
-	local env_val=$(tmux show-environment | grep --color=never "$env_name")
-	# The version below is still quite new for tmux. Uncomment this in the future :-)
-	#local env_val=$(tmux show-environment "$env_name" 2>&1)
-
-	if [[ ! $env_val =~ "unknown variable" ]]; then
-		local tmux_pwd=$(echo "$env_val" | sed 's/^.*=//')
-		echo "$tmux_pwd"
+MIN_MAJOR_VERSION="2"
+MIN_MINOR_VERSION="1"
+TMUX_VERSION="$(tmux -V)"
```

Comment:
I've searched the commti history with `git grep` and found out that `-V` was already present in v0.1 from November 2017 (see commit [0b832ea4](https://github.com/tmux/tmux/commit/0b832ea466cb013de2530f230d9a9ea9f1cd5641)). Or has the switch been removed in other versions?